src/processor/snapshot/mod.rs

```diff
@@ -17,13 +16,13 @@ use state_reconstruct_storage::{
     bytecode,
     types::{SnapshotFactoryDependency, SnapshotStorageLog},
 };
+use state_reconstruct_utils::{derive_final_address_for_params, h256_to_u256, unpack_block_info};
 use tokio::sync::mpsc;
 
 use super::Processor;
-use crate::util::{h256_to_u256, unpack_block_info};
 
 pub const DEFAULT_DB_PATH: &str = "snapshot_db";
-pub const SNAPSHOT_HEADER_FILE_NAME: &str = "snapshot-header.json";
+pub const SNAPSHOT_HEADER_FILE_NAME: &str = "snapshot_header.json";
```

Comment:
why? I bet I'll confuse the name when following some old doc, sooner or later...

Reply:
Every other snapshot file name uses `_` over `-` as a separator. I don't think we should deviate from that

Reply:
OK...